<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectFrontBackForm</class>
 <widget class="QDialog" name="SelectFrontBackForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="frontPres">
           <property name="text">
            <string>Front present</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Front mass number:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="frontA">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>260</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Front element number:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="frontZ">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>150</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Front width:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="frontW">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="frontUnit">
             <item>
              <property name="text">
               <string>mg/cm2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>g/cm2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>um</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="backPres">
           <property name="text">
            <string>Back present</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Back mass number:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="backA">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>260</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Back element number:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="backZ">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>150</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Back width:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="backW">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="backUnit">
             <item>
              <property name="text">
               <string>mg/cm2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>g/cm2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>um</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectFrontBackForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectFrontBackForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
